<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Events</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="ipAddresses">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>300</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="hosts">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>400</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelIpAddresses">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IP addresses:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHosts">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hosts:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="connections">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>721</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnections">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>400</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="payloads">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>721</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnection">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>373</y>
      <width>721</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
